config/core/test-configs.yaml

```diff
@@ -2257,6 +2257,18 @@ test_configs:
   - device_type: meson-gxl-s905x-libretech-cc
     test_plans:
       - baseline
+      - baseline-nfs
+      - kselftest-alsa
+      - kselftest-cpufreq
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-seccomp
+      - ltp-crypto
+      - ltp-fcntl-locktests
+      - ltp-ipc
+      # ltp-mm - Runs board out of memory
+      - ltp-pty
```

Comment:
Same problem as mentioned earlier:
https://staging.kernelci.org/test/plan/id/626139aeed990d896505f4d4/
https://storage.staging.kernelci.org/kernelci/staging-mainline/staging-mainline-20220421.0/arm64/defconfig/gcc-10/lab-broonie/ltp-pty-meson-gxl-s905x-libretech-cc.html

So it's not "OK" but the issue is not related to this PR.